lib/endpoints/class-wp-json-posts-controller.php

```diff
@@ -147,7 +147,9 @@ public function create_item( $request ) {
 		}
 
 		$post->post_type = $this->post_type;
+		add_filter( 'wp_insert_post_empty_content', '__return_false', 100 );
```

Comment:
Disabling parts of core code isn't something that we should do without really good reasons. The code here ensures that the data can't be changed to an invalid state: that is, posts must have *some* data (title, content, or excerpt) in the database to be valid.

@iseulde's original issue states:
> I know you can't update a post without the content, title AND excerpt empty, but you can leave two fields blank if one is filled. The API fails to do that.

That's the only issue that needs solving there: **updating** one field without needing to send the rest of the data (partial updates). This PR doesn't solve that issue.